selendroid-standalone/src/main/java/io/selendroid/standalone/android/impl/AbstractDevice.java

```diff
@@ -607,4 +607,31 @@ public String getModel() {
   public String getAPITargetType() {
     return apiTargetType;
   }
+
+  public void unlockScreen() throws AndroidDeviceException {
+    // Get phone's android version and whether the screen is off.
+    // Different ways to detect if the screen is off depending on the version.
+    CommandLine commandVersion = adbCommand("shell", "getprop", "ro.build.version.release");
+    String version = executeCommandQuietly(commandVersion);
+    CommandLine commandScreen = adbCommand("shell", "dumpsys", "power");
+    String screen = executeCommandQuietly(commandScreen);
+
+    // Kitkat
+    if (version.matches("^4\\.4(\\..*)?")) {
```

Comment:
No idea what the difference might be for versions before KitKat, but I'll try to find out.
If I can't figure it out, I'll do what you said.